deeplearning4j/deeplearning4j-scaleout/spark/dl4j-spark/src/main/java/org/deeplearning4j/spark/iterator/SparkAMDSI.java

```diff
@@ -106,7 +105,6 @@ public SparkAMDSI(MultiDataSetIterator iterator, int queueSize, BlockingQueue<Mu
 
     @Override
     protected void externalCall() {
-        TaskContextHelper.setTaskContext(context);
```

Comment:
This looks a bit suspicious. Why should externalCall be a noop here?

Reply:
It's out of date from spark. Somehow this was left over.